src/forms/forms-list.component.tsx

```diff
@@ -128,38 +135,46 @@ export default function FormsList(props: FormsListProps) {
           completedForms.map(encounter => {
             return (
               <div
-                role="button"
                 key={encounter.form.uuid}
                 tabIndex={-1}
                 style={{
                   borderBottom: "0.5px solid lightgray",
                   ...formItemStyle
                 }}
-                onClick={() =>
-                  handleFormSelected(
-                    encounter.form.uuid,
-                    encounter.form.name,
-                    encounter.uuid
-                  )
-                }
               >
-                <button className="omrs-btn omrs-text-action">
+                <button
+                  className="omrs-btn omrs-text-action"
+                  onClick={() =>
+                    handleFormSelected(
+                      encounter.form.uuid,
+                      encounter.form.name,
+                      encounter.uuid
+                    )
+                  }
+                >
                   {" "}
                   {encounter.form.name}{" "}
                 </button>
                 {/* <svg className="omrs-icon omrs-type-body-regular" fill="var(--omrs-color-ink-low-contrast)">
                   <use xlinkHref="#omrs-icon-chevron-right"></use>
                 </svg> */}
                 <br />
-                <span
-                  className="omrs-type-body-regular omrs-padding-left-24"
-                  style={{
-                    color: "var(--omrs-color-ink-medium-contrast)",
-                    fontFamily: "Work Sans"
-                  }}
-                >
-                  Completed on{" "}
-                  {encounter.encounterDateTime.toLocaleDateString()}{" "}
+                <span className="omrs-type-body-regular omrs-padding-left-24">
+                  <Link
+                    to={`${baseChartUrl}/encounter-viewer/${encounter.uuid}`}
+                    className="omrs-link omrs-filled-neutral"
+                  >
+                    <span
+                      style={{
+                        color: "var(--omrs-color-ink-medium-contrast)",
+                        fontFamily: "Work Sans"
+                      }}
+                    >
+                      {" "}
+                      Completed on{" "}
+                      {encounter.encounterDateTime.toLocaleDateString()}
```

Comment:
Isn't this supposed to be `encounterDatetime` as returned from the encounter json payload?

Reply:
That's being converted to javascript dat in the resource service

Reply:
https://github.com/AMPATH/ampath-esm-patient-chart-widgets/blob/master/src/openmrs-resource/encounter.resource.ts#L44